lib/configService.js

```diff
@@ -112,15 +113,26 @@ function processEnvironmentVariables() {
     ];
     const httpVariables = ['IOTA_HTTP_HOST', 'IOTA_HTTP_PORT', 'IOTA_HTTP_TIMEOUT', 'IOTA_HTTP_KEY', 'IOTA_HTTP_CERT'];
 
-    for (let i = 0; i < environmentVariables.length; i++) {
-        let value = process.env[environmentVariables[i]];
+    const protectedVariables = [
```

Comment:
Should an entry about the changes to be added in CHANGES_NEXT_RELEASE file?

Reply:
(Same for the "twin" PR)

Reply:
Fixed 68616bc